deployment/local-jvm/bash-scripts/killServices.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+pgrep java | awk '{system("kill "$1)}'
```

Comment:
This doesn't work on windows or WSL, and will kill everything Java surely?

Reply:
This crept in by me, I thought I had deleted it though. I couldn't work out a reasonable way to kill off all the Palisade JVM serivices

Reply:
I'll remove it.